app/assets/stylesheets/custom.scss

```diff
@@ -69,4 +69,12 @@ p{
 .container {
     display: flex;
     flex-direction: column;
+}
+
+.uncompleted-task {
+    color: red;
+}
+
+.completed-task {
+    color: green;
```

Comment:
File should end in a new line. There should be a setting on vs code to do this automatically